bots/tester/src/main/java/org/openjdk/skara/bots/tester/TestBot.java

```diff
@@ -88,38 +90,43 @@
                                          pr));
             } else {
                 // is there a job running for this PR?
+                var desc = pr.repository().name() + "#" + pr.id();
+                List<Job> jobs = List.of();
                 try {
-                    var jobs = ci.jobsFor(pr);
-                    if (!jobs.isEmpty()) {
-                        var shouldUpdate = false;
-                        for (var job : jobs) {
-                            if (!states.containsKey(job.id())) {
-                                shouldUpdate = true;
-                                states.put(job.id(), new Observation(job.state(), job.state()));
-                            } else {
-                                var observed = states.get(job.id());
+                    log.info("Getting test jobs for " + desc);
+                    jobs = ci.jobsFor(pr);
+                } catch (IOException e) {
+                    log.info("Could not retrieve test jobs for PR: " + desc);
+                    log.info(e.getMessage());
```

Comment:
Perhaps use log.throwing here?

Reply:
Ah, good point, wasn't aware of that one :+1: